networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -374,8 +374,8 @@ def descendants_at_distance(G, source, distance):
 
     Parameters
     ----------
-    G : NetworkX DiGraph
-        A directed graph
+    G : NetworkX Graph
```

Comment:
```suggestion
    G : NetworkX graph
```
lower case on "graph" removes confusion with the `Graph` object type.